assets/src/application/ui/forms/espressoForm/config.tsx

```diff
@@ -49,7 +49,7 @@ export const formConfig: EspressoFormProps = {
 					name: 'email',
 					label: 'Email',
 					fieldType: 'text',
-					htmlType: 'email',
+					type: 'email',
```

Comment:
this may have been this way to avoid a conflict with RFF... 
or is just to differentiate between the other "type" used in the form config (ie: fieldType)

Reply:
ok, adjusted back, thx